airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/StatusView.module.scss

```diff
@@ -46,4 +46,12 @@
   .contentCard {
     margin-bottom: 20px;
   }
+
+  .footer {
+    width: 100%;
+    height: 50px;
```

Comment:
:art: Minor nitpick: Usually we should try to avoid setting explicit `heights`. This comes with the problem that if any styling of elements changes inside we could end up overflowing this `height` at a later point. Usually in all cases you'd want to set a height to a container, you'll likely just want to apply `padding` or `margin` instead, to give spacing to the surrounding elements (or its contents in case of padding).

Reply:
That makes sense! Removed